package.json

```
@@ -3,7 +3,8 @@
 	"version": "2.0.0",
 	"description": "Unabashedly-barebones memoization library with an aim toward speed",
 	"type": "module",
-	"main": "dist/index.js",
+	"main": "dist/index.cjs",
+	"module": "dist/index.js",
```

I think it'd be best to keep the `type` and `main` formats in sync. Also, mentioned in passing in the previous pull request, `module` is non-standard. I think we could address both points using `exports` ([docs](https://nodejs.org/api/packages.html#conditional-exports)):

```suggestion
	"main": "dist/index.js",
	"exports": {
    "import": "./dist/index.js",
    "require": "./dist/index.cjs"
  },
```

Could you give this a try and see if it'd work in Gutenberg?

---

Yes I will.